cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ansible-modules"]
+= Automating management tasks using Ansible
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-ansible-modules
+toc::[]
+
+Ansible is an automation tool used to configure systems, deploy software, and perform rolling updates.
```

Comment:
I'm waiting to find out if "Ansible" is okay or if the shortname should be "Ansible Automation" which is how it's written in the spreadsheet of official product names/shortnames. I'm not sure if the list is out-of-date or if it's a new convention that we should start following.

Reply:
In RHV [documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/administration_guide/chap-automating_rhv_configuration_using_ansible) we use `Ansible`.

Reply:
Thanks for letting me know. I know that in the rest of the OpenShift docs, "Ansible" is used, but it's possible the "Ansible Automation" guideline is new. I'll let you know when I have confirmation.